src/UI/Microsoft.AspNetCore.Identity.UI.csproj

```diff
@@ -16,7 +16,6 @@
     <EmbeddedResource Remove="wwwroot/**/LICENSE*" />
     <None Remove="wwwroot/**/.bower.json" />
     <None Remove="wwwroot/**/LICENSE*" />
-    <None Include="THIRD-PARTY-NOTICES" Pack="true" />
```

Comment:
@javiercn This was added to include the THIRD-PARTY-NOTICES to the Identity UI package as part of https://github.com/aspnet/Identity/commit/4105ee67e907301539e0e2392e7ae50321b93fb5
But as a result it appears in all templates. 

Do we just want the notices in the GitHub repo or also as a part of the package? 
cc @Eilon @kichalla

Reply:
The notices file needs to be within the NUPKGs. But not in the generated project template or otherwise visible when this NUPKG is installed in a project.

Reply:
I have made a change to reflect that. It is in the nupkg but does not appear in the generated project